sktime/forecasting/model_selection/_split.py

```diff
@@ -67,6 +68,12 @@
 SPLIT_GENERATOR_TYPE = Iterator[SPLIT_ARRAY_TYPE]
 PANDAS_MTYPES = ["pd.DataFrame", "pd.Series", "pd-multiindex", "pd_multiindex_hier"]
 
+warnings.warn(
```

Comment:
if you want to avoid that this warning is raised when running crawling utility `all_estimator`, this should be added to the excepted module list there.

Also, replace "this module" by the full name, as users receiving the warning from an indirect import may get confused.

Reply:
Thank you for the feedback!
I've added the `_split` and `test_split` modules to the `EXCEPTED_MODULES` list in the `all_estimator` crawling utility as suggested and I've updated the deprecation warning to use the full module name using the `"__name__"` attribute for clarity.
Could you please review the updated changes?